Add batch test generator (#636)
aws-observability/aws-otel-test-framework: tools/batchTestGenerator/README.md

need to call out that I am assuming that EKSARM64 and EKS/EKS_FARGTE tests are ran on different clusters

> 

Reviewer: ```suggestion
```

# Batch Test Generator

> Creates batches of test cases for either local or use in a GitHub action.

Reviewer: ```suggestion

Creates batches of test cases for use locally or in a GitHub action.

```

## Commands
### local
Outputs a `test-case-batch` file.
### github
Sets the `batch-key` and `batch-values` output using std out. 

## Flags
### Globally Available flags
#### --testCaseFilePath

> Path to test case file.

Reviewer: Is this the output file?  Would `--output` make sense here?  It's a common flag name and shorter than the alternative.

Author: No this is the `testcases.json` file not the final `test-case-batch` file.

#### --include
List of services to include. If not provided will default to all.
#### --eksarm64amp
Endpoint for EKS ARM 64 AMP workspace.
#### --eksarm64cluster
Cluster name for EKS ARM 64 tests.
#### --eksarm64region
Region for EKS ARM 64 tests.
#### --eksamp

> Endpoint for EKS ARM 64 AMP workspace.

Reviewer: ```suggestion
Endpoint for EKS AMP workspace.
```
